libraries/net/node.cpp

```diff
@@ -4145,6 +4161,25 @@ namespace graphene { namespace net { namespace detail {
       trigger_p2p_network_connect_loop();
     }
 
+   void node_impl::add_seed_node(const std::string& endpoint_string)
+   {
+      VERIFY_CORRECT_THREAD();
+      std::vector<fc::ip::endpoint> endpoints;
+      try
+      {
+         endpoints = graphene::net::node::resolve_string_to_ip_endpoints(endpoint_string);
+      }
+      catch(...)
+      {
+         elog( "Unable to resolve endpoint during attempt to add seed node ${ep}", ("ep", endpoint_string) );
```

Comment:
1. `wlog` is better;
2. logging in this file instead of `application.cpp` means they'll be saved to `p2p.log`, so we'll no longer see them in console nor `default.log`, not a big deal though.